public/robots.txt

```diff
@@ -1,5 +1,11 @@
 # See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
 #
 # To ban all spiders from the entire site uncomment the next two lines:
-# User-agent: *
-# Disallow: /
+User-agent: *
+Disallow: /
+
+User-agent: *
+Disallow: /namespaces
+Disallow: /repositories
+Disallow: /teams
+Disallow: /admin/
```

Comment:
The `/admin/` has an extra slash. It should be just `/admin` I think.

Moreover, this is missing:

- `/errors`
- `/search`
- `/help`
- `/team_users`
- `/v2`
- `/tags`
- `/users`
- `/toggle_enabled`
- `/application_tokens`
- `/documentation`
- `/api`